applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -121,10 +123,9 @@ int hw_codec_volume_adjust(int8_t adjustment)
 	 */
 	adjustment *= 2;
 
-	int16_t volume_reg_val;
+	uint32_t volume_reg_val;
```

Comment:
Need to take into account that adjustment can be negative. Line:134 will roll over if volume_reg_val < adjustment and adjustment is negative